### DISCUSSION TITLE
Image File attachment add in message input box

### DISCUSSION THREAD

**Discussion by iyyappanrock535** [chatscope/chat-ui-kit-react#73] [Show and tell]
Image File attachment add in message input box. How to add that image file in the message input box?

**Comment by supersnager** [comment]
@iyyappanrock535 Thank you for your question. Do you want to display an image in the **&lt;MessageInput /&gt;** before sending?

**Reply by iyyappanrock535** [reply]
Yes yes

**Comment by ztanruan** [comment]
Any updates regarding this request ? - preview image on message input container before sending message

**Reply by sclea** [reply]
Hey!

I did it that way:

![image](https://github.com/chatscope/chat-ui-kit-react/assets/15073709/d19d659f-643e-49dc-97e5-f7ca44b6c01b)

Above is previous message sent, and in the input box how it looks during editing.
Text and imgs may follow in any order:

![image](https://github.com/chatscope/chat-ui-kit-react/assets/15073709/52efc0a9-49e8-4ed9-ad19-043edaadadc6)

My custom styles to style img in the message list and in the input:

> .cs-message-input__content-editor img {
    height: 25px;
    margin-right: 3px;
    margin-left: 3px;
    vertical-align: middle;
}
.cs-message__content-wrapper img {
	width: 100%;
	margin-top: 8px;
	margin-bottom: 4px;
}
`

It's not as cool as say telegram or whatsapp although so here's the way to grow ^)

**Comment by aragop25** [comment]
Do you guys have code to show how to let a user attatch a file upon clicking the attatch button? I want to be able to open the file browser when user clicks on the attatch button

**Reply by aragop25** [reply]
@sclea
